server/tests/publish/belga_newsml_1_2_tests.py

```diff
@@ -2414,3 +2414,387 @@ def test_related_article_belga_archive_text(self):
+        "first_name": "John",
+        "last_name": "Smith",
+        "role": ObjectId("5d542206c04280bc6d6157f9"),
+    },)
+
+    roles = ({
+        "_id": ObjectId("5d542206c04280bc6d6157f9"),
+        "author_role": "AUTHOR",
+        "editor_role": "AUTHOR"
+    },)
+
+    vocabularies = (
+        {
+            "_id": "belga-coverage-new",
+            "field_type": "custom",
+            "items": [],
+            "type": "manageable",
+            "schema": {
+                "name": {},
+                "qcode": {},
+                "parent": {}
+            },
+            "service": {
+                "all": 1
+            },
+            "custom_field_type": "belga.coverage",
+            "display_name": "belga coverage new",
+            "unique_field": "qcode",
+        },
+    )
+
+    subscriber = {
+        '_id': 'some_id',
+        'name': 'Dev Subscriber',
+    }
+
+    @mock.patch('superdesk.publish.subscribers.SubscribersService.generate_sequence_number', lambda s, sub: 1)
+    @mock.patch('belga.search_providers.BelgaCoverageSearchProvider.api_get', lambda self,
+                endpoint, params: belga_apiget_response)
+    def setUp(self):
+        init_app(self.app)
+        self.app.data.insert('users', self.users)
+        self.app.data.insert('archive', self.archive)
+        self.app.data.insert('roles', self.roles)
+        self.app.data.insert('vocabularies', self.vocabularies)
+        # insert pictures
+        media_items = (
+            {
+                '_id': 'pic_1',
+                'content': BytesIO(b'pic_one_content'),
+                'content_type': 'image/jpeg',
+                'metadata': {
+                    'length': 10
+                }
+            },
+            {
+                '_id': 'pic_2',
+                'content': BytesIO(b'pic_two_content'),
+                'content_type': 'image/jpeg'
+            },
+            {
+                '_id': 'pic_3',
+                'content': BytesIO(b'pic_three_content'),
+                'content_type': 'image/jpeg',
+                'metadata': {
+                    'length': 12
+                }
+            },
+            {
+                '_id': 'audio_1',
+                'content': BytesIO(b'the man who sold the world'),
+                'content_type': 'audio/mp3',
+                'metadata': {
+                    'length': 12
+                }
+            },
+            {
+                '_id': 'video_1',
+                'content': BytesIO(b'czech rap xD'),
+                'content_type': 'video/mp4',
+                'metadata': {
+                    'length': 12
+                }
+            },
+            {
+                '_id': 'pdf_1',
+                'content': BytesIO(b'it is a pdf'),
+                'content_type': 'application/pdf',
+                'metadata': {
+                    'length': 40248
+                }
+            },
+        )
+        for media_item in media_items:
```

Comment:
We only need to put video here, don't we?

small note: There is extra 1.2 https://github.com/superdesk/superdesk-belga/blob/0b6f39ea9b033e193d4b96d350b9230a26e0caa1/server/belga/publish/belga_newsml_1_2.py#L37

Reply:
Yes, it's copy-paste issue :) Only video.